activestorage/lib/tasks/activestorage.rake

```diff
@@ -2,12 +2,23 @@ require "fileutils"
 
 namespace :activestorage do
   desc "Copy over the migration needed to the application"
-  task :install do
-    migration_file_path = "db/migrate/#{Time.now.utc.strftime("%Y%m%d%H%M%S")}_active_storage_create_tables.rb"
-    FileUtils.mkdir_p Rails.root.join("db/migrate")
-    FileUtils.cp File.expand_path("../../active_storage/migration.rb", __FILE__), Rails.root.join(migration_file_path)
-    puts "Copied migration to #{migration_file_path}"
+  task install: :environment do
+    destination = Rails.root.join("db/migrate")
+    FileUtils.mkdir_p(destination) unless File.exist?(destination)
```

Comment:
Why is the existence check necessary? `FileUtils.mkdir_p` won’t raise an error if the given directory already exists.